src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml

```diff
@@ -1,6 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0014</DiagnosticId>
+    <Target>M:System.Console.SetWindowSize(System.Int32,System.Int32):[T:System.Runtime.Versioning.UnsupportedOSPlatformAttribute]</Target>
+    <Left>net8.0/mscorlib.dll</Left>
+    <Right>net9.0/mscorlib.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0014</DiagnosticId>
+    <Target>P:System.Console.WindowHeight:[T:System.Runtime.Versioning.UnsupportedOSPlatformAttribute]</Target>
+    <Left>net8.0/mscorlib.dll</Left>
+    <Right>net9.0/mscorlib.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0014</DiagnosticId>
+    <Target>P:System.Console.WindowWidth:[T:System.Runtime.Versioning.UnsupportedOSPlatformAttribute]</Target>
+    <Left>net8.0/mscorlib.dll</Left>
+    <Right>net9.0/mscorlib.dll</Right>
+  </Suppression>
```

Comment:
How was this generated? I tried as described in https://github.com/dotnet/runtime/pull/75824#issuecomment-1303714861 but didn't get any changes related to Console.
cc @ericstj

Reply:
@ViktorHofer

Reply:
`dotnet.cmd src/libraries/apicompat --no-dependencies /p:ApiCompatGenerateSuppressionFile=true` should work, make sure you've built libs before doing that.

I think the comment you linked to had a different property name.  The one listed in the error message and shared here should be the right one: `ApiCompatGenerateSuppressionFile`
https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#apicompatgeneratesuppressionfile

Reply:
Also these are all CP0014 - they are what I'd expect for the attribute changes to the reference assembly.  If you're OK with those changes then these are fine (so long as build passes they are in sync).

Reply:
Thanks for checking them out. Btw, this is what I get from running dotnet.cmd:
```
C:\git\runtime>dotnet.cmd src\libraries\apicompat --no-dependencies /p:ApiCompatGenerateSuppressionFile=true
C:\Program Files\dotnet
Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET program, but dotnet-src\libraries\apicompat does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
```